src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,14 @@
+import logo from '../../assets/logo.png';
+import ProfileId from './ProfileId';
+import Styles from '../../styles/FolderPage.module.css';
+
+const NavBar = () => {
+  return (
+    <div className={Styles.Nav}>
+      <img className={Styles.logo} src={logo} alt='logo' />
+      <ProfileId />
```

Comment:
P1:
profileId라는 이름보다 무난한 이름이 좋을 것 같아요.
```suggestion
      <Profile />
```